packages/components/src/custom-select-control-v2/stories/index.story.tsx

```diff
@@ -44,6 +45,9 @@ const meta: Meta< typeof CustomSelectControlV2 > = {
 			</div>
 		),
 	],
+	args: {
+		onChange: fn(),
+	},
```

Comment:
I don't think that adding these lines of code is necessary since we're already specifying `argTypesRegex` above — I tested removing these lines on my machine and I was still getting "actions" firing correctly. This also matches the [documentation about actions](https://storybook.js.org/docs/essentials/actions#automatically-matching-args).

Given that we typically use the `argTypesRegex` way, I'm not sure we even need to list `@storybook/test` as a dependency for the time being?

Reply:
Unfortunately, we seem to need these, otherwise I'm getting an error when navigating to these 3 stories (I tested them all, one by one, manually):

* http://localhost:50240/?path=/docs/components-customselectcontrol-v2--docs
* http://localhost:50240/?path=/docs/components-tabpanel--docs
* http://localhost:50240/?path=/docs/components-tabs--docs

Here's the error:

![Screenshot 2024-12-11 at 16 40 07](https://github.com/user-attachments/assets/9fb08537-d397-4f10-b3bd-ac4733555d1c)

Once I add the action spies, the errors disappear.

Reply:
Weird. It doesn't make sense to me, since I would then expect this error to appear on a lot more stories. I think the checks a bit buggy (and [we may not be the only ones](https://github.com/storybookjs/storybook/discussions/28314)). 

For the sake of moving forward with the refactor, I'm OK keeping those changes and investigate the reason for the error in a separate follow-up PR

Reply:
I agree, and yep, would like to take a look separately. 

Opened an issue here: https://github.com/WordPress/gutenberg/issues/67834